kin-sdk/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -292,6 +294,27 @@ class KinAccountIntegrationTest {
         assertThat(kinAccountReceiver.balanceSync.value(), equalTo(BigDecimal("20.00000")))
     }
 
+    @Test
+    @LargeTest
+    @Throws(Exception::class)
+    fun sendTransaction_Success() {
+        val (kinAccountSender, kinAccountReceiver) = onboardAccounts(senderFundAmount = 100)
```

Comment:
what's the purpose of this test?

Reply:
Didn't so one, only happy path with memo so this one is without memo